plugins/search/src/components/SearchBar/SearchBar.tsx

```diff
@@ -14,14 +14,71 @@
  * limitations under the License.
  */
 
-import React, { ChangeEvent, useEffect, useState } from 'react';
+import React, { useEffect, KeyboardEvent, useState } from 'react';
 import { useDebounce } from 'react-use';
 import { InputBase, InputAdornment, IconButton } from '@material-ui/core';
 import SearchIcon from '@material-ui/icons/Search';
 import ClearButton from '@material-ui/icons/Clear';
 
 import { useSearch } from '../SearchContext';
 
+type PresenterProps = {
+  value: string;
+  onChange: (value: string) => void;
+  onClear?: () => void;
+  onSubmit?: () => void;
+  className?: string;
+  placeholder?: string;
+};
+
+export const SearchBarBase = ({
+  value,
+  onChange,
+  onSubmit,
+  className,
+  placeholder,
+}: PresenterProps) => {
+  const onKeyDown = React.useCallback(
+    (e: KeyboardEvent<HTMLInputElement>) => {
+      if (onSubmit && e.key === 'Enter') {
+        onSubmit();
+      }
+    },
+    [onSubmit],
+  );
+
+  const handleClear = React.useCallback(() => {
+    onChange('');
+  }, [onChange]);
+
+  return (
+    <InputBase
+      data-testid="search-bar-next"
+      fullWidth
+      placeholder={placeholder ?? 'Search in Backstage'}
```

Comment:
Oh yeah!  Since we make it possible for integrators to brand their backstage instance to their liking, we should probably use the configured `app.title` instead of the literal string `Backstage`

Reply:
Ah yes, I was wondering if there existed such a configuration. Thanks for pointing that out!

Fixed in 8dfae42.